policy/modules/system/init.if

```diff
@@ -3482,6 +3500,23 @@ interface(`init_manage_all_unit_files',`
 	manage_lnk_files_pattern($1, systemdunit, systemdunit)
 ')
 
+#########################################
+## <summary>
+##	Template to allow init to link to a key ring.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain whose key ring init should be allowed to link.
+##	</summary>
+## </param>
+#
+template(`init_link_key_template',`
+	gen_require(`
+		type init_t;
+	')
+	allow init_t $1:key link;
```

Comment:
that is just an interface and not a template: init_link_keyrings()

Reply:
o actually that is not allowed this way. Youll have to address this some other way

AFAIK param domain ($1) is only allowed as a source, not as a target

Reply:
OK. I think we can use an attribute for this.